src/views/splash/components/admin.jsx

```diff
@@ -0,0 +1,44 @@
+var React = require('react');
```

Comment:
In the case where you need React for the jsx syntax, disable the require line like so https://github.com/LLK/scratch-www/blob/633fee162aa81665e539f06cfcf46151cd2c22d6/src/lib/render.jsx#L4 — there's not a good way to let the linter know about jsx transpilation yet.